nautobot/core/runner/runner.py

```diff
@@ -238,9 +239,22 @@ def run_app(**kwargs):
             )
             return
 
+        # Prompt user to enable installation metrics
+        installation_metrics_enabled = None
+        while installation_metrics_enabled is None:
+            installation_metrics_prompt = input(
+                """Would you like to allow Nautobot to send anonymous Nautobot version and plugin version metrics? [y/n]: """
```

Comment:
Might be good to include a link to the published docs (when available) for more information? Non blocking.